backpack/extensions/secondorder/diag_ggn/activations.py

```diff
@@ -17,3 +21,23 @@ def __init__(self):
 class DiagGGNTanh(DiagGGNBaseModule):
     def __init__(self):
         super().__init__(derivatives=TanhDerivatives())
+
+
+class DiagGGNELU(DiagGGNBaseModule):
+    def __init__(self):
+        super().__init__(derivatives=ELUDerivatives())
+
+
+class DiagGGNSELU(DiagGGNBaseModule):
+    def __init__(self):
+        super().__init__(derivatives=SELUDerivatives())
+
+
+class DiagGGNLeakyReLU(DiagGGNBaseModule):
+    def __init__(self):
+        super().__init__(derivatives=LeakyReLUDerivatives())
+
+
+class DiagGGNLogSigmoid(DiagGGNBaseModule):
+    def __init__(self):
+        super().__init__(derivatives=LogSigmoidDerivatives())
```

Comment:
Do we have tests for these? I'm just confused because the PR says only `ConvNd` and `ConvTransposeNd`are extended

Reply:
Umm, actually we dont have it here yet. We have it for `diag-h`

Reply:
Added in local settings